site2/docs/sql-getting-started.md

```diff
@@ -117,18 +117,45 @@ You can query the mock data.
 If you want to query your own data, you need to ingest your own data first. You can write a simple producer and write custom defined data to Pulsar. The following is an example. 
 
 ```java
-public class Test {
-    
-     public static class Foo {
+public class TestProducer {
+
+    public static class Foo {
```

Comment:
Use  `@Data`  and  `@NoArgsConstructor` instead of implement get and set methods maybe better.

Reply:
That's a demo program for others to copy. They may not use `lombok`, so i think use `set get` more friendly